### DISCUSSION TITLE
IntelliJ Plugin

### DISCUSSION THREAD

**Discussion by szymon-romanko** [codewars/codewars.com#2843] [Ideas]
It would be cool to see some IDE integration so that you could use your favourite code editor instead of the onsite built-in.
I'm thinking of a plugin that will allow you to see details of the kata directly in your IDE and an attempt button, so you can test without copying code to the website.
Tell me what you think:)

**Comment by QBERT18** [comment]
Hey. Yeah it would be nice. But i am not sure if you can get all the needed data from the api https://dev.codewars.com/#code-challenges-api
But surely there are ways to crawl the challenge pages for more information.
